src/main/java/com/incognia/api/IncogniaAPI.java

```diff
@@ -395,23 +395,136 @@ public TransactionAssessment registerPayment(RegisterPaymentRequest request)
    * }</pre>
    *
    * @param feedbackEvent type of feedback event
-   * @param timestamp Instant when the fraud or event happened
+   * @param occurredAt Instant when the fraud or event happened
    * @param identifiers the user's identifiers
    * @throws IncogniaAPIException in case of api errors
    * @throws IncogniaException in case of unexpected errors
    */
   public void registerFeedback(
-      FeedbackEvent feedbackEvent, Instant timestamp, FeedbackIdentifiers identifiers)
+      FeedbackEvent feedbackEvent, Instant occurredAt, FeedbackIdentifiers identifiers)
       throws IncogniaException {
-    registerFeedback(feedbackEvent, timestamp, identifiers, false);
+    registerFeedback(feedbackEvent, occurredAt, identifiers, false);
   }
 
+  /**
+   * Shares feedback about a risk decision, improving the quality of risk assessments. Check <a
+   * href="https://dash.incognia.com/api-reference#operation/feedbacks-post">the docs</a><br>
+   * Example:
+   *
+   * <pre>{@code
+   * IncogniaAPI api = new IncogniaAPI("client-id", "client-secret");
+   * try {
+   *      Instant timestamp = Instant.now();
```

Comment:
maybe we shouldn't use `Instant.now()` in the docs, since this should be the ATO timestamp

Reply:
Fixed